src/extension/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "module": "commonjs",
     "target": "es6",
     "outDir": "out",
-    "lib": ["es6", "dom"],
+    "lib": ["es2017", "dom"],
```

Comment:
would this change possibly affect anything else?

Reply:
I changed this to use the padStart method, not sure if it will affect anything else.. @jcoc611-microsoft @piotrmark any idea?